spec/graphql/relay/node_spec.rb

```diff
@@ -9,43 +9,18 @@
   end
 
   describe ".field" do
-    describe "with custom resolver" do
-      it "executes the custom resolve instead of relay default" do
-        id = "resolver_is_hardcoded_so_this_does_not_matter"
-
-        result = star_wars_query(%|{
-          nodeWithCustomResolver(id: "#{id}") {
-            id,
-            ... on Faction {
-              name
-              ships(first: 1) {
-                edges {
-                 node {
-                   name
-                   }
-                }
-              }
-            }
-          }
-        }|)
+    it 'accepts a custom definition' do
+      faction = StarWars::DATA['Faction'][1]
 
-        expected = {"data" => {
-          "nodeWithCustomResolver"=>{
-            "id"=>"RmFjdGlvbi0x",
-            "name"=>"Alliance to Restore the Republic",
-            "ships"=>{
-              "edges"=>[
-                {"node"=>{
-                    "name" => "X-Wing"
-                  }
-                }
-              ]
-            }
-          }
-        }}
-
-        assert_equal(expected, result)
+      node_field = GraphQL::Relay::Node.field do
+        name "nod3"
+        description "The Relay Node Field"
+        resolve ->(_, _ , _) { faction }
       end
+
+      node_field.name = "nod3"
+      node_field.description = "The Relay Node Field"
```

Comment:
I noticed these assign the same values as assigned above, are these meant to be assertions?